mne/channels/channels.py

```diff
@@ -434,29 +434,15 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         :func:`mne.viz.plot_trans`.
 
         .. versionadded:: 0.12.0
-
         """
         from ..viz.utils import plot_sensors
         return plot_sensors(self.info, kind=kind, ch_type=ch_type, title=title,
                             show_names=show_names, ch_groups=ch_groups,
                             axes=axes, block=block, show=show)
 
+    @copy_function_doc_to_method_doc(anonymize_info)
     def anonymize(self):
-        """Anonymize measurement information in place by removing
-        'subject_info', 'meas_date', 'file_id', 'meas_id' if they exist in
-        ``info``.
-
-        Returns
-        -------
-        self : instance of Raw | Epochs | Evoked
-            The data container.
-
-        Notes
-        -----
-        Operates in place.
-
-        .. versionadded:: 0.13.0
-        """
+        """.. versionadded:: 0.13.0"""
```

Comment:
Scratch that, looks like there is another you can get rid of:

https://github.com/wmvanvliet/mne-python/blob/b89390b3033b8c6e4f71f2fcacb76fcae6773db6/mne/io/base.py